Add KGO Dataplane documentation (#1384)
Kong/developer.konghq.com: app/operator/konnect/reference/migrate-1.4-1.5.md

---
title: Migrate Konnect DataPlanes from {{ site.operator_product_name }} 1.4 to 1.5
description: "How do I migrate from {{ site.operator_product_name }} 1.4 to 1.5, taking in to account breaking changes?"
content_type: reference
layout: reference
products:
  - operator
breadcrumbs:
  - /operator/
  - index: operator
    group: Konnect
  - index: operator
    group: Konnect
    section: Reference

---

This document helps to migrate from the `gateway-operator.konghq.com` to the `konnect.konghq.com` `KonnectExtension`.

1. Label the certificate `Secret`:

    ```bash
    kubectl label secret -n kong konnect-client-tls konghq.com/konnect-dp-cert=true
    ```

1. Install new kubernetes-configuration CRDs:

    ```bash
    kustomize build https://github.com/kong/kubernetes-configuration/crd/gateway-operator | kubectl apply --server-side -f -
    ```

    <!-- Requires https://github.com/Kong/docs.konghq.com/pull/8588 to merge -->

>     To verify version compatibility with kubernetes-configuration CRDs, please consult the [version compatibility table][version_compat].

Reviewer: ```suggestion
    To verify version compatibility with kubernetes-configuration CRDs, please consult the [version compatibility table](/operator/reference/version-compatibility/#kubernetes-configuration-crds).
```

>     > NOTE: In case CRDs were installed via `helm` you might need to force conflicts:

Reviewer: ```suggestion
    {:.info}
    > Note: In case CRDs were installed via `helm` you might need to force conflicts:
```

    ```bash
    kustomize build https://github.com/kong/kubernetes-configuration/crd/gateway-operator | kubectl apply --server-side --force-conflicts -f -
    ```

>     [version_compat]: https://docs.konghq.com/gateway-operator/1.5.x/reference/version-compatibility/#kubernetes-configuration-crds

Reviewer: ```suggestion
```
